system-test/spanner.ts

```diff
@@ -1844,6 +1894,219 @@ describe('Spanner', () => {
       });
     });
 
+    describe('protoMessage', () => {
+      before(async function () {
+        if (IS_EMULATOR_ENABLED) {
+          this.skip();
```

Comment:
Why skip for emulator ?

Reply:
There are issues in emulator which is not yet addressed. Will unskip the tests in a seperate PR when emulator is ready.